hdx_hapi/endpoints/util/endpoint_code_generator.py

```diff
@@ -0,0 +1,510 @@
+#!/usr/bin/env python
```

Comment:
Since the code generator supports HAPI but is not really part of it, as in, HAPI is not dependent on it to run, could we move it to another folder ? Maybe we could have a root folder called `tools` ?